src/pages/user/InstitutionPresentationCard.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { InstitutionUnit } from '../../types/institution.types';
+import { selectInstitutionNameByLanguage } from '../../utils/helpers';
+import { Button } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import InstitutionSelector from './InstitutionSelector';
+
+const StyledSelectedInstitution = styled.div`
+  margin-top: 0.5rem;
+  padding-top: 0.5rem;
+  padding-left: 0.5rem;
+  background-color: ${({ theme }) => theme.palette.background.default};
+  height: 5rem;
+`;
+
+const StyledInstitutionText = styled.div`
+  height: 1.5rem;
+`;
+
+const StyledInstitutionTextMain = styled(StyledInstitutionText)`
+  font-weight: bold;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const StyledButton = styled(Button)`
+  margin: 0.5rem;
+`;
+
+interface InstitutionPresentationProps {
+  institutionUnit: InstitutionUnit;
+  addNewInstitutionUnit: (cristinUnitId: string) => void;
+}
+
+const InstitutionPresentation: React.FC<InstitutionPresentationProps> = ({
+  institutionUnit,
+  addNewInstitutionUnit,
+}) => {
+  const [selectedCristinUnitId, setSelectedCristinUnitId] = useState(institutionUnit?.cristinUnitId || '');
+  const [openEdit, setOpenEdit] = useState(false);
+  const { t } = useTranslation('profile');
+
+  const handleConfirm = () => {
+    setOpenEdit(false);
+    addNewInstitutionUnit(selectedCristinUnitId);
+  };
+
+  return (
+    <>
```

Comment:
jeg tror dette kan struktureres litt annerledes (spesielt med tanke på hvordan datamodellen vil se ut). hvis vi lar være å bruke level1name og level2name, og heller tenker at hver institution kan ha en institution, så kan vi introdusere rekursiv rendering. 

Dette er ikke noe som må gjøres nå, men kan være verdt å tenke på det til fremtiden.
Demo her: https://medium.com/@swatisucharita94/recursive-rendering-in-react-42666102eae2

Reply:
Hadde tenkt å gå løs på dette etter denne pr'en